src/atmos_spectral/driver/solo/mixed_layer.F90

```diff
@@ -595,13 +595,13 @@ subroutine mixed_layer (                                               &
 endif
 
 if(update_albedo_from_ice) then
-    call read_ice_conc(Time_next)
-    land_ice_mask=.false.
-    where(land_mask.or.(ice_concentration.gt.ice_concentration_threshold))
-        land_ice_mask=.true.
-    end where
+  call read_ice_conc(Time_next)
+  land_ice_mask=.false.
+  where(land_mask.or.(ice_concentration.gt.ice_concentration_threshold))
+    land_ice_mask=.true.
+  end where
 else
-    land_ice_mask=land_mask
+  land_ice_mask=land_mask
 endif
 
```

Comment:
This is uncommented in @lqxyz 's branch except line 604. I have reverted the comment in the PR as in the group meeting we saw no reason why it should be (if I recall).